avalon/tools/loader/app.py

```diff
@@ -1,20 +1,18 @@
 import sys
 import time
 
-from ..projectmanager.widget import (
-    AssetWidget,
-    AssetModel,
-    preserve_selection,
-)
+from ..models import AssetModel
+from ..widgets import AssetWidget
+from ..lib import preserve_selection
 
 from ...vendor.Qt import QtWidgets, QtCore
 from ... import api, io, style
 from .. import lib
 
-from .lib import (
-    refresh_family_config,
-    refresh_group_config,
-    get_active_group_config,
+from ..lib import (
```

Comment:
No, this should be `from .. import lib` which you're already doing above, but then not using?

Reply:
It's being imported three times (!) in three different ways, let's clean this up.

Reply:
Thanks @mottosso - well spotted! I tried to simplify and clean things up with [this commit](https://github.com/getavalon/core/pull/440/commits/5de72f38cf1ac5a3b39f7f5e6e2729fbc1a95255). Let me know if that suffices.